docs/core/project-sdk/msbuild-props.md

```diff
@@ -665,7 +665,7 @@ The `CodeAnalysisTreatWarningsAsErrors` property lets you configure whether code
 ### EnforceCodeStyleInBuild
 
 > [!NOTE]
-> This feature is currently experimental and may change between the .NET 5 and .NET 6 releases.
+> This feature is currently experimental.
```

Comment:
@mavasani Is it still considered experimental?

Reply:
@jmarolf @mikadumont

Reply:
with this issue resolved: https://github.com/dotnet/roslyn/issues/49044 I am not aware of any issues. folks who subsume  preview SDKs may get errors due to the codestyle version they use being newer then what is included in Visual Studio but I believe we are fine with this experience. for RTM sdks and Visual Studios this feature should be fully useable

Reply:
We should also remove experimental from Code Analysis property page:

![image](https://user-images.githubusercontent.com/10605811/141108839-31fbba8a-f4d2-4e8b-8e45-79a4043a5592.png)